backend/api/src/Routes/routes.go

```diff
@@ -36,18 +36,19 @@ func SetupRoutes(gormObj *gorm.DB) (*gin.Engine, error) {
 	r.POST("/api/login", login)
 	r.GET("/api/list-exams", listExams)
 	r.GET("/api/list-news", listNews)
-	r.GET("/api/list-due-exams", listDueExams)
 	// auth protected routes
 	auth := r.Group("/", authMiddleware)
 	{
 		auth.GET("/api/auth-hello", hello)
 		auth.GET("/api/list-user-exams", listUserExams)
+		auth.POST("/api/refresh-jwt", refreshJWT)
 		auth.POST("/api/register-exam", registerToExam)
 		auth.POST("/api/unregister-exam", unregisterFromExam)
 		auth.POST("/api/add-user-expo-push-token", addUserExpoPushToken)
 		adminAuth := auth.Group("/", adminMiddleware)
 		{
 			adminAuth.GET("/api/list-exam-users", listExamUsers)
+			adminAuth.GET("/api/list-due-exams", listDueExams)
```

Comment:
Why should due exams be put into admin protected routes?

Reply:
It is only used as an API Route for testing purposes. Could even be removed. But just put it in admin routes instead.